src/views/document/RouteView.vue

```diff
@@ -77,7 +77,12 @@
 
         <div class="box">
           <markdown-section :document="document" :field="fields.summary" />
-          <markdown-section :document="document" :field="fields.route_history" />
+          <markdown-section v-if="locale.route_history" :document="document" :field="fields.route_history" />
+          <div v-else-if="showMissingHistoryBanner" class="notification is-info no-print">
+            <edit-link :document="document" :lang="lang" show-even-if-not-logged>
+              History is missing, please provide it if you have information.
+            </edit-link>
+          </div>
```

Comment:
Couldt it be more consistent if the notifcation box was displayed below a history section? Currently, there is no history section.

Reply:
I do not know. I would say that it is only an advertising for editing, not an information for the random reader: maybe with a History section it uses too much space ?

Reply:
Ok. Let's try it this way and see if there is room for improvement later on.